spec/controllers/prompts_controller_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'spec_helper'
+
+describe PromptsController do
+  include LoginMacros
```

Comment:
I'd suggest also including `RedirectExpectationHelper` and then using that in all the examples that check that something redirects with an error.